crates/bvs-vault-base/src/msg.rs

```diff
@@ -113,43 +113,43 @@ pub enum VaultQueryMsg {
 /// Not exported.
 /// This is just a wrapper around `Uint128`, so that the schema can be generated.
 #[cw_serde]
-struct SharesResponse(Uint128);
+pub struct SharesResponse(Uint128);
```

Comment:
```suggestion
pub struct SharesResponse(Uint128);
```

It's stated above that it's not exported, you need to update the comment if you plan to export.

Reply:
Valid, will fix.

Reply:
can consider `pub(crate)` if you just want these to be visible for the crate and not exported 🤔